### PR TITLE
Rework Docker tutorial

### PR DESCRIPTION
**Pull request by tomdee**

### CODE REVIEW

File: master/getting-started/docker/tutorials/basic.md [hunk 1]
```diff
@@ -77,84 +77,52 @@ The Calico IPAM driver provides address assignment that is geared towards
...
-
-    docker network create --driver calico --ipam-driver calico-ipam net1
-    docker network create --driver calico --ipam-driver calico-ipam net2
-    docker network create --driver calico --ipam-driver calico-ipam net3
-
-#### b) Networking using Calico IPAM in a cloud environment
-
-For Calico IPAM in a cloud environment that doesn't enable direct container to
-container communication (DigitalOcean, GCE), you need to first create a Calico
+This can be changed e.g. to choose a different IP range or to enabled IPIP, by creating a Calico
```
Comment (djosborne) [inline master/getting-started/docker/tutorials/basic.md]: typo: `to enable`

**Comment by djosborne** [comment]
LGTM. Really excited to finally see simpler IPAM info here.

File: master/getting-started/docker/tutorials/basic.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ automatically add the container to the Calico network, creating all necessary
 Calico configuration and setting up the interface and routes in the container
 accordingly.
 
-The Calico IPAM driver may be used in addition to the the Calico network
+The Calico IPAM driver must be used in addition to the the Calico network
 driver.  This provides IP address management using the configured Calico IP
 Pools as address pools for the container, preferentially selecting sub-blocks
 of IPs for a particular host.
```
Comment (robbrockbank) [inline master/getting-started/docker/tutorials/basic.md]: Below::  you will to set up -> you will need to set up

**Comment by djosborne** [comment]
Merging now so these changes are included in next imminent RC. @tomdee has agreed to open another PR to correct these typos

**Pull request merged**